### PR TITLE
rephrasing project 1 description

### PR DESCRIPTION
**Pull request by AmoebaProtozoa**
Signed-off-by: AmoebaProtozoa <davidyangad@gmail.com>

Related issue: #282

### What's changed

Restructured the project 1 guide according to course structure proposed at #275 

Stripped out unnecessary details for the project and move them inside quoted hints.

### CODE REVIEW

**Review by JGEthanChen — COMMENTED** [review]
Nice work!  While considering the format should be similar to other project docs, how about dividing into 3 parts?
1. Overview just behind the title without sub-title;
2. `The Code` including your `Related Files`;
3. Implement steps.

Furthermore, I think it need more detail about  **Put/Delete/Get/Scan** operations and **column family**. Would you mind improving that with me?

**Comment by AmoebaProtozoa** [comment]
> Nice work! While considering the format should be similar to other project docs, how about dividing into 3 parts?
> 
> 1. Overview just behind the title without sub-title;
> 2. `The Code` including your `Related Files`;
> 3. Implement steps.
> 
> Furthermore, I think it need more detail about **Put/Delete/Get/Scan** operations and **column family**. Would you mind improving that with me?

Hi @Whorra! That sounds great! I'll restructure the doc later this week. 

Feel free to add anything you see fit to this PR (like more details about server operations or column family). Let me know if you need access to the fork repo.

**Comment by JGEthanChen** [comment]
@AmoebaProtozoa  Okay! It would be great we can finish the doc together. ^_^

File: doc/project1-StandaloneKV.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 
-In this project, you will build a standalone key/value storage [gRPC](https://grpc.io/docs/guides/) service with the support of the column family. Standalone means only a single node, not a distributed system. [Column family]( <https://en.wikipedia.org/wiki/Standard_column_family> ) (it will abbreviate to CF below) is a term like key namespace, namely the values of the same key in different column families is not the same. You can simply regard multiple column families as separate mini databases. It’s used to support the transaction model in the project4, you will know why TinyKV needs the support of CF then.
+In this project, you will build a mini key-value storage service with the following characteristics.
 
-The service supports four basic operations: Put/Delete/Get/Scan. It maintains a simple database of key/value pairs. Keys and values are strings. `Put` replaces the value for a particular key for the specified CF in the database, `Delete` deletes the key's value for the specified CF, `Get` fetches the current value for a key for the specified CF, and `Scan` fetches the current value for a series of keys for the specified CF.
+#### Standalone Key-value storage
```
Comment (tisonkun) [inline doc/project1-StandaloneKV.md:5]: Header level is malformed.

File: doc/project1-StandaloneKV.md [hunk 2]
```diff
@@ -1,8 +1,25 @@
 # Project1 StandaloneKV
 
-In this project, you will build a standalone key/value storage [gRPC](https://grpc.io/docs/guides/) service with the support of the column family. Standalone means only a single node, not a distributed system. [Column family]( <https://en.wikipedia.org/wiki/Standard_column_family> ) (it will abbreviate to CF below) is a term like key namespace, namely the values of the same key in different column families is not the same. You can simply regard multiple column families as separate mini databases. It’s used to support the transaction model in the project4, you will know why TinyKV needs the support of CF then.
+In this project, you will build a mini key-value storage service with the following characteristics.
```
Comment (tisonkun) [inline doc/project1-StandaloneKV.md:3]: Why do you change standalone to "mini"?

File: doc/project1-StandaloneKV.md [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
 
-The service supports four basic operations: Put/Delete/Get/Scan. It maintains a simple database of key/value pairs. Keys and values are strings. `Put` replaces the value for a particular key for the specified CF in the database, `Delete` deletes the key's value for the specified CF, `Get` fetches the current value for a key for the specified CF, and `Scan` fetches the current value for a series of keys for the specified CF.
+#### Standalone Key-value storage
+
+Standalone means only a single node, not a distributed system. This StandaloneKV in Project1 is just for storing key/value pairs locally and its underlying storage capability is built upon a pure go key-value database called badgerDB.
```
Comment (tisonkun) [inline doc/project1-StandaloneKV.md:7]: Please use "key-value" consistently.

Comment (tisonkun) [reply doc/project1-StandaloneKV.md:7]: Also "Standalone means only a single node, not a distributed system" sounds in Chinglish. @dcalvin do you have an idea expressing this?

File: doc/project1-StandaloneKV.md [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+
+#### Support Column Family
+
+[Column family]( <https://en.wikipedia.org/wiki/Standard_column_family> ) (abbreviate to CF below) denotes a logical layer between key-value entry and the database. You can consider CF as a key namespace, and multiple CFs are usually implemented as almost separate databases. Different values can be set for the same key in different CFs (CF store has some transactional property that is necessary for our transaction model in project 4).
```
Comment (tisonkun) [inline doc/project1-StandaloneKV.md:11]:
```suggestion
[Column family](https://en.wikipedia.org/wiki/Standard_column_family) (abbreviate to CF below) denotes a logical layer between key-value entry and the database. You can consider CF as a key namespace, and multiple CFs are usually implemented as almost separate databases. Different values can be set for the same key in different CFs (CF store has some transactional property that is necessary for our transaction model in project 4).
```

**Comment by tisonkun** [comment]
@zhangyangyu could you please also take a look at this PR?

**Pull request closed without merging**